### ISSUE TITLE
Training Code

### ISSUE THREAD

**Issue by Strike1999** [Westlake-AGI-Lab/Distill-Any-Depth#5]
Great work!

When will the training code be released?

Thanks!

**Comment by shuiyued** [comment]
Hello, 
Thank you for your interest in our work. 
The release timeline for the training code is related to the progress of our paper.

**Comment by icoz69** [comment]
If you are interested in the loss being used, you may refer to HDN repo at: https://github.com/icoz69/HDN

**Comment by fairy003** [comment]
> Hello, Thank you for your interest in our work. The release timeline for the training code is related to the progress of our paper.

Hi~ I'm wondering when the training code will be released, especially regarding the multi-teacher training strategy.

**Comment by zengqi0730** [comment]
> Hello, Thank you for your interest in our work. The release timeline for the training code is related to the progress of our paper.

When will the training code be released?
